src/modes/stack/app/src/AddNode.svelte

```diff
@@ -42,10 +42,10 @@
       place: "Internal",
     };
     return console.log(newNode);
-    stack.update((s) => ({
-      network: s.network,
-      nodes: [...s.nodes, newNode],
-    }));
+    // stack.update((s) => ({
+    //   network: s.network,
+    //   nodes: [...s.nodes, newNode],
+    // }));
```

Comment:
not sure why this is commented out on this PR

Reply:
Hello @kevkevinpal I have removed the commented code